resources/cosmetics/cosmetics.json

```
@@ -103,6 +104,23 @@
     "AAIiAAAAAAAAAAAAAAAAAAAAAIDD8YnweTiiD5FIYEIgEpkIRCKBCoFIpCIQeTwyPB6RjEAkEIgQKEQiApFAIEIgEYkIOAKfCIGIIyIAAAAAAAAAAAA": {
       "name": "openfront",
       "role_group": "creator"
+    },
+
+    "APFYAAAAAAD4BwAA_A8AAOwPAAD8DwAA_A8AAHwAAAD8AwACfgAAAn8AAMb_AQDufwEA_H8AAPx_AAD4PwAA8D8AAOAdAADACAAAQAgAAMAYAAAAAAAAAAAAAAAAAAAAAAAA":{
```

These should be cropped so there's only a 1px border.
```suggestion
    "AJlMAAAAAP8A8D8A9gfA_wD4HwAfAOAfAn5A4A8Y_wf3v8D_B_j_AP4PgP8B4B0AGAEAIQBgDAAAAAA":{
```